app/code/Magento/Analytics/Model/ReportXml/ModuleIterator.php

```diff
@@ -34,6 +34,7 @@ public function __construct(
      *
      * @return array
      */
+    #[\ReturnTypeWillChange]
     public function current()
```

Comment:
I would recommend to specify a proper return type for this method instead

Reply:
Hi @fascinosum , in PHP >= 8 the return type for this method is _mixed_  (https://www.php.net/manual/en/class.iteratoriterator.php) , but in PHP 7.4 this return type is not specified yet.